USFMConverter/Core/Util/FileSystem.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace USFMConverter.Core.Util
+{
+    public static class FileSystem
+    {
+        public static ICollection<FileInfo> GetFilesInDir(FileInfo dir)
```

Comment:
Why does this function exist? It is just a very small wrapper around GetFiles that doesn't add any value it just makes things more complicated.